PF-build.sh

```diff
@@ -110,74 +157,223 @@ if ! type zip > /dev/null; then
 		echo "Run git Bash under Administrator privilege and"
 		echo "navigate to the directory /c/Program Files/Git/mingw64/bin,"
 		echo "you can run the command $(tput setaf 2)ln -s /c/Program Files/7-Zip/7z.exe zip.exe$(tput sgr0)"
-		exit
+		exit 3
 	elif [ $OSTYPE == "linux-gnu" ]; then
 		echo "$(tput setaf 1)Missing 'zip' which is important to run this script"
 		echo "install it with the command $(tput setaf 2)'sudo apt-get install zip'$(tput sgr0)"
-		exit
+		exit 3
+	fi
+fi
+# Check python ... needed during language build
+if ! type python > /dev/null; then
+	if [ $OSTYPE == "msys" ]; then
+		echo "$(tput setaf 1)Missing 'python' which is important to run this script"
+		exit 4
+	elif [ $OSTYPE == "linux-gnu" ]; then
+		echo "$(tput setaf 1)Missing 'python' which is important to run this script"
+		echo "As Python 2.x will not be maintained from 2020 please,"
+		echo "install it with the command $(tput setaf 2)'sudo apt-get install python3'."
+		echo "Check which version of Python3 has been installed using 'ls /usr/bin/python3*'"
+		echo "Use 'sudo ln -sf /usr/bin/python3.x /usr/bin/python' (where 'x' is your version number) to make it default.$(tput sgr0)"
+		exit 4
 	fi
 fi
-###End prepare bash enviroment
 
-BUILD_ENV="1.0.2"
+#### End prepare bash environment
+
+
+#### Set build environment 
+ARDUINO_ENV="1.8.5"
+BUILD_ENV="1.0.6"
+BOARD="PrusaResearchRambo"
+BOARD_VERSION="1.0.1"
+BOARD_URL="https://raw.githubusercontent.com/DRracer/Arduino_Boards/master/IDE_Board_Manager/package_prusa3d_index.json"
+BOARD_FILENAME="prusa3drambo"
+BOARD_FILE_URL="https://raw.githubusercontent.com/DRracer/Arduino_Boards/master/IDE_Board_Manager/prusa3drambo-1.0.1.tar.bz2"
+PF_BUILD_FILE_URL="https://github.com/3d-gussner/PF-build-env/releases/download/$BUILD_ENV/PF-build-env-$BUILD_ENV.zip"
+LIB="PrusaLibrary"
 SCRIPT_PATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
 # List few useful data
 echo
-echo "Script path:" $SCRIPT_PATH
-echo "OS         :" $OS
-echo "OS type    :" $OSTYPE
+echo "Script path :" $SCRIPT_PATH
+echo "OS          :" $OS
+echo "OS type     :" $OSTYPE
+echo ""
+echo "Ardunio IDE :" $ARDUINO_ENV
+echo "Build env   :" $BUILD_ENV
+echo "Board       :" $BOARD
+echo "Specific Lib:" $LIB
 echo ""
 
-#### Start prepare building
+#### Start prepare building environment
 
 #Check if build exists and creates it if not
 if [ ! -d "../build-env" ]; then
-    mkdir ../build-env || exit 2
+    mkdir ../build-env || exit 5
```

Comment:
Could this folder be named so that it's apparent these are downloaded files for building Prusa Firware? For example "PF-build-dl" or something like that...

Reply:
Sure everything is possible ... I used `build-env` as the `build.sh` does the same. Wanted to be as close as possible to the `build.sh` script and` PF-build-env` repository by @mkbel

Reply:
Changed to `PF-build-dl`